projects/fmcomms2/kc705/README.md

```diff
@@ -0,0 +1,10 @@
+# FMCOMMS2 HDL Project
+
+## Building the project
+
+Corresponding device tree: [kc705_fmcomms2-3.dts](https://github.com/analogdevicesinc/linux/blob/main/arch/microblaze/boot/dts/kc705_fmcomms2-3.dts)
```

Comment:
```suggestion
Corresponding device trees:

- for FMCOMMS2/3: [kc705_fmcomms2-3.dts](https://github.com/analogdevicesinc/linux/blob/main/arch/microblaze/boot/dts/kc705_fmcomms2-3.dts)
- for FMCOMMS4: [kc705_fmcomms4.dts](https://github.com/analogdevicesinc/linux/blob/main/arch/microblaze/boot/dts/kc705_fmcomms4.dts)
```
The same should be done for the rest of the READMEs (to add the device tree for the FMCOMMS4 board as well)